mautic/mautic
# [UI] Nav tabs and nav content

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR makes tab contents act like a layer, differentiating them from the background.






| Before                                 | After
| -------------------------------------- | ---
|    ![image](https://github.com/mautic/mautic/assets/116097999/c253e8b5-77e7-4408-9a86-66a081eb0c1f) | ![image](https://github.com/mautic/mautic/assets/116097999/6977cf0b-f9ac-4a54-9a61-287816e87be4)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Details > Have a look
3. Open Plugins > Click to open any and see
